### ISSUE TITLE
File does not exist .nwm_version

### ISSUE THREAD

**Issue by danrosen25** [NCAR/wrf_hydro_nwm_public#572]
#### Issue ####
utils/CMakeLists.txt references .nwm_version file when NWM_META=1 but this file does not exist. This causes CMake to crash.

https://github.com/NCAR/wrf_hydro_nwm_public/blob/385b7971febb28d48685b1da3d39e27eb89b7156/trunk/NDHMS/utils/CMakeLists.txt#L7

#### Instructions ####
Build WRF-Hydro using CMake and set environment variable NWM_META=1

**Comment by rcabell** [comment]
This file only exists on tags that correspond to NWM releases. Is there a way in CMake to give a fallback value if the file doesn't exist?

**Comment by danrosen25** [comment]
@rcabell What should we set the NWM_VERSION to if the file is missing?
```
if(EXISTS "../.nwm_version")
   file (STRINGS "../.nwm_version"  NWM_VERSION) 
else()
   TBD
endif()
```

**Comment by rcabell** [comment]
@danrosen25 what's it being used for? Would setting it to `undefined` break anything?

**Comment by danrosen25** [comment]
As far as I can tell it won't break anything. NWM_VERSION sets nwmVersion, get_nwm_version, and get_model_version but they are all character (len=512). I can give this a try?

**Comment by rcabell** [comment]
@danrosen25 yes, give that a try -- I think it's a simpler work-around than always having that file present/defined (given the current release workflow)

**Comment by danrosen25** [comment]
Fix merged into 5.2.x branch with #577
